ldk/go/filesystemService.go

```diff
@@ -2,25 +2,54 @@ package ldk
 
 import (
 	"context"
+	"os"
 	"time"
 
 	"github.com/open-olive/loop-development-kit/ldk/go/proto"
 )
 
 // FilesystemService is an interface that defines what methods plugins can expect from the host
 type FilesystemService interface {
-	Dir(context.Context, string) ([]FileInfo, error)
+	Dir(context.Context, string) ([]os.FileInfo, error)
 	ListenDir(context.Context, string, ListenDirHandler) error
-	File(context.Context, string) (FileInfo, error)
 	ListenFile(context.Context, string, ListenFileHandler) error
+	Open(context.Context, string) (File, error)
+	Create(context.Context, string) (File, error)
+	MakeDir(context.Context, string, uint32) error
+	Copy(context.Context, string, string) error
+	Move(context.Context, string, string) error
+	Remove(context.Context, string, bool) error
 }
 
 type FileInfo struct {
```

Comment:
Is this the same `FileInfo` that `Stat` returns or different? Might be a bit confusing?

Reply:
Its the same